src/main/java/uk/ac/cam/cl/dtg/isaac/api/AssignmentFacade.java

```diff
@@ -973,6 +985,12 @@ public Response assignGameBoard(@Context final HttpServletRequest request,
                         .toResponse();
             }
 
+            if (null != assignmentDTOFromClient.getNotes() && assignmentDTOFromClient.getNotes().length() > 240) {
```

Comment:
I think we should move this out to a segue constant, like `MAX_NOTE_CHAR_LENGTH = 240` and then use that `.toString()` in the error response, so we don't end up with this 240/ 500 discrepancy.